.github/workflows/build.yml

```diff
@@ -22,10 +25,18 @@ jobs:
                 symfony: ["^5.4", "^6.4"]
                 persistence: ["^2.0", "^3.0"]
                 collections: ["^1.8", "^2.0"]
+                experimental: [false]
+                app_env: ["test"]
                 include:
                     - php: "8.1"
                       pagerfanta: "^3.7"
                       symfony: "^5.4"
+                      app_env: "test"
+                      experimental: false
+                    - php: 8.3
+                      symfony: "^7.0"
+                      app_env: "test_without_hateoas"
+                      experimental: true
```

Comment:
Similar to GridBundle, as we would like to add support for SF7 with this PR, I'm not sure if this flag is needed, wdyt? 🤔

Reply:
yes it is a problem.
https://github.com/willdurand/BazingaHateoasBundle/issues/108